manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -148,7 +150,7 @@ To reduce the time and effort needed to incorporate data into models, best pract
 
 **4. Focus on uncertainty**
 
-Understanding the uncertainty of forecasts is crucial to interpreting and understanding their utility. Models used for forecasting should be probabilistic to properly quantify uncertainty and to convey how this uncertainty increases through time. Evaluation of forecast models should include assessment of how accurately they quantify uncertainty as well as point estimates  [@hooten2015].
+Understanding the uncertainty of forecasts is crucial to interpreting and understanding their utility. Models used for forecasting should be probabilistic to properly quantify uncertainty and to convey how this uncertainty increases through time. Evaluation of forecast models should include assessment of how accurately they quantify uncertainty as well as point estimates  [@hooten2015; q@harris2018].
```

Comment:
one more thing, typo for the "@harris2018" citation.

Reply:
Argh. Must of done that accidentally while. Looking at things. Thanks. Fixed locally.